.github/workflows/code_checks.yml

```diff
@@ -106,9 +103,6 @@ jobs:
       - name: Cleanup
         if: always()
         run: |
-          docker rmi -f vets-api:${{ env.DOCKER_IMAGE }} 2> /dev/null || true
-          docker rm -f ${{ env.DOCKER_IMAGE }}-postgres-1 2> /dev/null || true
-          docker network prune -f
           rm -rf .git
```

Comment:
I think "Cleanup" can be removed completely. This was added because of issue with self-hosted runners